mw-webapp/vite.config.ts

```diff
@@ -1,11 +1,21 @@
-import {defineConfig} from "vite";
+import {defineConfig, loadEnv} from "vite";
 import react from "@vitejs/plugin-react";
 import eslint from "vite-plugin-eslint";
 import viteTsconfigPaths from "vite-tsconfig-paths";
 
 export default defineConfig(() => {
+  const env = loadEnv("", process.cwd(), "");
+
   return {
     build: {outDir: "build"},
     plugins: [react(), eslint(), viteTsconfigPaths()],
+    define: {
```

Comment:
Could you add some helper with signature like: const getEnvs = (envs: string[]): Record<your types here>, currently too much copypaste

Reply:
I have made changes to the code, but I am unsure if they are correct.